vehicle/mkv/software/lvbms/lvbms.c

```diff
@@ -0,0 +1,67 @@
+#include "lvbms_config.h"
+#include "libs/adc/api.h"
+#include "libs/gpio/api.h"
+#include "libs/timer/api.h"
+#include "libs/spi/api.h"
+#include "vehicle/mkv/software/lvbms/can_api.h" // TODO: why?? 
```

Comment:
This is so that you can access the functions defined in the CAN API like `can_send_lvbms`.